schema_based_on_diagram.sql

```diff
@@ -38,9 +38,17 @@ create table invoice_items(
     foreign key (treatment_id) REFERENCES treatments(id)
 )
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221997241-353a0af4-fc54-4f00-9fbd-fb409198a546.png)

**[Optional]**

- [ ] It would be nice if you Acknowledge someone in this section :blush: It shows your connections with other people and the organizations :thinking: You could have acknowledged Microverse, your coding partner or the person who has given you this project or the person whom design you are using :+1: Kindly acknowledge someone in this section to make your readme file descriptive and professional :+1:

Reply:
Hi @HAMMAS-SALEEM , thanks for the review, the optional suggestions will be implemented.